api/app/controllers/gardens_controller.rb

```diff
@@ -44,6 +44,34 @@ def nurture
     end
   end
 
+  def journal
+    @garden = Garden.find(params[:garden_id])
```

Comment:
We're repeating a lot this initialization, maybe we can have a `before_action` for those methods that are doing that, something like

` before_action :set_garden, only: %i[show...`
 
`  def set_garden
    @garden = Garden.find(params[:id] || params[:garden_id])
  end
`